### PR TITLE
Optimize a little bit the menu bar and Charlotte's pr

### PR DESCRIPTION
**Pull request by ScottCTD**
- Like the title said.

### CODE REVIEW

File: Application/gui_main.py [hunk 1]
```diff
@@ -625,37 +616,31 @@ def init_menu(self) -> None:
...
         covid_marker.setCheckable(True)
-        closure_marker.setCheckable(True)
-
         covid_marker.setChecked(True)
-        closure_marker.setChecked(True)
-
-        covid_marker.setStatusTip('Show marker in COVID - 19 plot')
-        closure_marker.setStatusTip('Show marker in Closure plot')
-
+        covid_marker.setStatusTip('Show markers in the COVID-19 Cases plot')
```
Comment (CharlotteChenyy) [inline Application/gui_main.py:627]: 这里是不是该和statusbar一样 改成'Show/Hide markers in the COVID-19 Cases plot'

File: Application/gui_main.py [hunk 2]
```diff
@@ -625,37 +616,31 @@ def init_menu(self) -> None:
...
         covid_marker.triggered.connect(self.toggle_marker_covid)
+
+        closure_marker = QAction('Closure Plot', self)
+        closure_marker.setCheckable(True)
+        closure_marker.setChecked(True)
+        closure_marker.setStatusTip('Show markers in the School Closure Status plot')
```
Comment (CharlotteChenyy) [inline Application/gui_main.py:633]: Same. 'Show/Hide markers in the School Closure Status plot'

**Review by CharlotteChenyy — APPROVED** [review]
Scott牛👍👍👍

**Comment by ScottCTD** [comment]
ok 没别的问题直接merge就行

im gonna sleeeeeeeeeeep

**Pull request merged**